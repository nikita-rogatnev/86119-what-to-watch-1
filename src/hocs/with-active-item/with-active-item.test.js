import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import withActiveItem from "./with-active-item";

Enzyme.configure({adapter: new Adapter()});

it(`withActiveItem renders correctly`, () => {
  const tree = shallow(<withActiveItem/>);

  expect(toJson(tree)).toMatchSnapshot();
});
