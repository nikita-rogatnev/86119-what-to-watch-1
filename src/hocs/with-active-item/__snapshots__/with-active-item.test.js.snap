// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveItem renders correctly 1`] = `<withActiveItem />`;
